crates/modelardb_storage/src/delta_lake.rs

```diff
@@ -504,3 +504,159 @@ impl DeltaLake {
         format!("{}/{METADATA_FOLDER}/{table_name}", self.location)
     }
 }
+
+/// Functionality for transactionally writing [`RecordBatches`](RecordBatch) to a Delta Table stored
```

Comment:
It seems that sometimes we write `Delta Table` and sometimes we write `delta table`. Elsewhere in the repository I have used `Delta table` so maybe change all occurrences in this file to do the same.

Reply:
Fixed by writing `Delta table` everywhere in the file.